src/lib.rs

```diff
@@ -525,7 +525,7 @@ impl Filter {
             // Up to 95% occupancy
             // 19/20 == 0.95
             // Overflow is not possible here as it'd have overflowed in the constructor.
-            self.total_buckets().get() * 19 / 20
+            ((self.total_buckets().get() * 19) as f64 / 20.0).ceil() as u64
```

Comment:
I would prefer to fix the bug inside the initial calculation of qbits resp max_qbits:
- the floating point math in capacity has some performance impact
- it's no longer only filling up to 95%

Reply:
I can certainly fix it in the constructor as well, but I will have to leave this calculation or we will run into the same edge cases when it re-runs the capacity calculation. 

That being the case, I think I will look to move this calculation to be a stored value in the struct so that it can just be referenced rather than recalculated on every call.

Reply:
```suggestion
            ((1 << self.max_qbits.unwrap_or(self.qbits).get()) * 19).div_ceil(20)
```

We can fix it with integer operations, it's only an additional `add`

Reply:
Actually, it's more than one `add` because optimizing compiler are transforming the division into a multiplication + power of 2 divisions. But I suppose it's still plenty fast.

Reply:
Okay awesome, I believe I've updated that logic if you believe it will be fast enough now using the integer based calculations 😄